UPGRADE.md

```diff
@@ -33,11 +36,17 @@ Some keys have been removed from [color palette](https://github.com/CartoDB/cart
 - disabledOpacity
 - selectedOpacity
 - focusOpacity
-- other, all removed but tooltip and divider
+- other, all removed but divider, which is moved to first level
+
+Some others have been moved or replaced because they aren't native MUI keys and are so specific to some components, these are:
+
+- charts (replaced by `theme.palette.black[%]`)
+
+`grey palette` is not used to design (and therefore not for style) components directly. We have a set of neutral colors to use in the custom `default` variant.
 
-Some others have been moved because they aren't native MUI keys and are so specific to some components, these are:
+We also have a set of `shade` colors (with transparency):
 
-- charts
-- other
+- black
+- white
 
-These sets of keys are now in: `componentsPalette`.
+Important: `primary.relatedLight` and `secondary.relatedLight` has to be replaced by `primary.background` and `secondary.background`.
```

Comment:
I think that these details are fine, cause are useful for the 'UPGRADE' process itself, but we have also some "mixed" comments (?) about internal changes / refactors (a bit at the beginning of the doc?)... Maybe it's easier to have a separated section in the document talking about 'internals' (some notes useful for carto-react developers) vs 'upgrade' advices for carto-react users (just interested in external / api changes). Just a feeling, not something urgent now, but to think about

Reply:
Oh, good point. I'll give it a thought to see how to apply it to the document. Thanks for the suggestion :)